index.html

```diff
@@ -25,6 +25,7 @@ <h2>Feedback en Issues</h2>
   </section>
   <section id="sotd"></section><!-- Wordt automatisch gevuld -->
   <section data-include-format="markdown" data-include="sections/ch01.md"></section>
+  <section data-include-format="markdown" data-include="sections/Extensie%20geo-objecten.md"></section>
```

Comment:
Haal de spatie weg uit het bestand en noem het `extensie-geo-objecten.md` zodat hij correct wordt toegevoegd aan Respec. Nu komt hij niet voor in de preview.